pyprophet/scoring/runner.py

```diff
@@ -255,6 +255,50 @@ def print_summary(self, result):
             logger.opt(raw=True).info("\n")
 
 
+class PyProphetMultiLearner(PyProphetRunner):
+    """
+    Implements the learning and scoring workflow for PyProphet with multiple classifiers run sequentially
+    """
+
+    def run_algo(self, part=None):
+        """
+        Runs the learning and scoring algorithm for multiple classifiers.
+
+        Returns:
+            tuple: A tuple containing the result, scorer, and weights.
+        """
+        if self.glyco:
+            raise click.ClickException(
+                "Multi-classifier learning is not supported for glycopeptide workflows."
+            )
+        else:
+            config_lda = self.config.copy()
+            config_lda.runner.classifier = "LDA"
+                    
+            # remove columns that are not needed for LDA
+            table_lda = self.table.drop(columns=["var_precursor_charge", "var_product_charge", "var_transition_count"], errors='ignore')
+ 
+            (result_lda, scorer_lda, weights_lda) = PyProphet(config_lda).learn_and_apply(table_lda)
```

Comment:
Will this run the full learning and scoring on the ss num iters and xval num iters, and then do a second pass with XGBoost with the same data for ss num iters and xval num iters? I am wondering if this results in any over fitting?